OrbitGl/App.cpp

```diff
@@ -635,7 +576,7 @@ void OrbitApp::AddSamplingReport(
   auto report = std::make_shared<SamplingReport>(sampling_profiler);
 
   for (SamplingReportCallback& callback : app->m_SamplingReportsCallbacks) {
-    callback(report);
+    callback(app->CreateOrGetDataView(DataViewType::CALLSTACK), report);
```

Comment:
We use GetOrCreateXYZ elsewhere in the code, consider using the same pattern for consistency.  "Get" is before "Create" since it's the typical use case.

Reply:
done.